java-frontend/src/main/java/org/sonar/java/UCFGJavaVisitor.java

```diff
@@ -87,6 +87,8 @@ public UCFGJavaVisitor(File workdir) {
     this.protobufDirectory = new File(new File(workdir, "ucfg"), "java");
     if(!protobufDirectory.exists()) {
       protobufDirectory.mkdirs();
+    } else {
+      index = protobufDirectory.list().length;
```

Comment:
I would appreciate a comment here to explicitly mention that we restart from the number of UCFG already created.